sdk/appconfiguration/app-configuration/samples/react-sample/src/reportWebVitals.ts

```diff
@@ -0,0 +1,15 @@
+import { ReportHandler } from 'web-vitals';
+
+const reportWebVitals = (onPerfEntry?: ReportHandler) => {
```

Comment:
We probably don't need this for the sample, and you can delete the import, the call to `reportWebVitals` and the big comment that goes with it in index.tsx